manim/camera/moving_camera.py

```diff
@@ -15,11 +15,14 @@
 from ..mobject.frame import ScreenRectangle
 from ..mobject.types.vectorized_mobject import VGroup
 from ..utils.color import WHITE
+from typing import Optional, Union
+from ..mobject.mobject import Mobject
+import numpy as np
 
 
 # TODO, think about how to incorporate perspective
 class CameraFrame(VGroup):
-    def __init__(self, center=ORIGIN, **kwargs):
+    def __init__(self, center: np.ndarray = ORIGIN, **kwargs):
```

Comment:
this can be also a `List`, a better type hint might be `Sequence[float]` as can be seen from this discussion here:
https://github.com/ManimCommunity/manim/issues/1337